text/0000-fabric-rsa-alg.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: default
+title: RSA Algorithm Support
+nav_order: 3
+---
+
+- Feature Name: Re-introduce RSA crypto support
+- Start Date: 2020-11-11
+- RFC PR: 
+- Fabric Component: bccsp, crypto
+- Fabric Issue: 
+
+# Summary
+[summary]: #summary
+
+Fabric had RSA crypto feature implemented since the very beginning even if there was a mismatch between the code and the documentation.
+With the introduction of Fabric 2.x the RSA code was removed, so currently existing production ledgers with transactions signed by RSA no longer could be validated after upgrade to the new version.  Original RSA support commits:
```

Comment:
Given the code references cited above, I don't see how transactions could be signed and verified with RSA.